app/models/api_credential.rb

```diff
@@ -37,6 +37,19 @@ def is_refresh_token_expired?
     refresh_token_expires_at < Time.current
   end
 
+  # clear tokens
+  # token argument takes in two strings: api_token and refresh_token
+  def revoke_token(token)
```

Comment:
Mind separating these out like a `revoke_api_token` and `revoke_refresh_token` function

Reply:
Initially this is what I wanted to do but then I thought of avoiding duplication
Let me know if you still want to keep them separate.

Reply:
I have separated it into two functions.